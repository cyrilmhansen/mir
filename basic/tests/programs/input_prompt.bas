10 INPUT "How many apples? ";N
20 PRINT N
