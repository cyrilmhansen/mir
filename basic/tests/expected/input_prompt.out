How many apples? 5
